MdiChild/MdiChild.cpp

```diff
@@ -16,7 +17,14 @@ void MdiChild::NewFile(QSize CanvasSize)
     isUntitled = true;
     openedFile = tr("drawing%1.jpg").arg(sequenceNumber++);
     setWindowTitle(openedFile + "[*]");
-    this->setFixedSize(CanvasSize);
+    setMinimumSize(QSize(1,1));
+    setMaximumSize(QSize(99999, 99999));
```

Comment:
setMinimumSize & setMaximumSize is redundant here. The default values set in .ui file are enough.

Reply:
Yes, but we don't have any .ui file for MdiChild